UM/Qt/Bindings/OutputDeviceManagerProxy.py

```diff
@@ -27,7 +27,10 @@ def __init__(self, parent = None) -> None:
         self._device_manager.activeDeviceChanged.connect(self._onActiveDeviceChanged)
         self._onActiveDeviceChanged()
 
+        self._manualDeviceInfo = {}
```

Comment:
I'd expect MyPy to complain about this missing type documentation.

Reply:
Fixed!